cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -154,32 +145,30 @@ public Collection<StructuredRow> multiRead(Collection<? extends Collection<Field
   /**
    * Creates a SELECT query that fetches rows from a given set of keys.
    *
-   * @param keyFields a map from field name to set of field values to query
+   * @param multiKeys a map from field name to set of field values to query
```

Comment:
Please explain that this has OR with nest AND otherwise it's hard to understand why is it a nested collection

Reply:
Updated, thanks for the review!